Source/UI/Delegates/CountrySelectionViewControllerDelegate.swift

```diff
@@ -7,5 +7,5 @@ public protocol CountrySelectionViewControllerDelegate: AnyObject {
     ///
     /// - parameter country: Country selected
     /// - parameter regionCode: Region code of the selected country
-    func onCountrySelected(country: Country)
+    func onCountrySelected(country: Country, tag: Int)
```

Comment:
we have the ordered list of cells from `DefaultBillingFormViewModel.init`. Does it make more sense to pass around the updated row index or can we rely on the order from there instead?